java/jdbc/ConnectionSamples/ADBQuickStart.java

```diff
@@ -96,36 +94,43 @@ public static void main(String args[]) throws Exception {
     // connections allowed on the connection pool.
     pds.setMaxPoolSize(20);
 
+
     // Get the database connection from UCP.
     try (Connection conn = pds.getConnection()) {
       System.out.println("Available connections after checkout: "
           + pds.getAvailableConnectionsCount());
       System.out.println("Borrowed connections after checkout: "
           + pds.getBorrowedConnectionsCount());
+          
       // Perform a database operation
-      doSQLWork(conn);
+      try {
```

Comment:
There's no need for nested `try` blocks, you can do it with just one.

Reply:
Done